### DISCUSSION TITLE
December Monthly Challenge

### DISCUSSION THREAD

**Discussion by aurelieverrot** [Virtual-Coffee/VC-Community-Docs#218] [Monthly Challenges]
After three highly demanding challenges in a row, and with the Holidays period approaching, it seems like we want something more smooth, cosy, and chocolatey 😊

Please share your ideas for December!

**Comment by aurelieverrot** [comment]
Why not help someone make their first stream where they make crepes? (lol, it's a challenge Nerando gave me)


Another thing I have in mind, but not specifically for the challenge:
One thing we did last year was the Advent of Code, which was absolutely not smooth and cosy 😅  though it was something more-or-less (or completely) supported by Mike, and I thought that even if it is not the official monthly challenge, this is something we might want to keep.
Even if we don't solve all of them, we can still organize zoom meetings to solve them as a group, or things like that.
https://adventofcode.com/2021

**Reply by BekahHW** [reply]
We do have the code-challenges channel. Mike used to do a lot of the organizing around that. Maybe a message in that channel to see who's interested would be helpful.

**Comment by ClJarvis** [comment]
Chocolate recipe  challenge? Lighting talk dessert.

**Comment by ClJarvis** [comment]
I think we talked about doing an advent of code challenge. Taking one of them and working in co working room I bet slack ate that discussion.
UPDATE: I checked slack ate the conversation. But It was suggested taking a  one of the advent challenges and going over solutions in a lunch and learn or co working session.

**Comment by BekahHW** [comment]
I like advent of code and I think it could be cool, but I wonder how much engagement or how "burnt out" ppl feel. 

If we have the support for advent of code with this team, I think that's great and we can go for it. 

I, personally, don't have the bandwidth to participate at all--which is totally fine.

I know @tkshill had some ideas about making this a recovery month. We could focus on connecting with community--it doens't have to be our community, but maybe members own communities. Month of care, community, connection. I'm just throwing out some alliteration now. lol

**Reply by aurelieverrot** [reply]
I think it is better to have a chill month as the main challenge. December is already stressful enough to add more workload on it.

Some people will do the advent of code anyway, we have a channel ready, we can just say "hey it's there" and let people organize themselves. And everything can be async, last year there was one post per day, and people could do it at their own speed.



What if the challenge is to do just one thing over the month with the theme of Holidays or Winter. 
The title would be: "look what a software engineer does for the Holiday / Winter season". And we let people choose if it's code, or design, or craft, or cooking. The goal is just to have something to post somewhere: an article, a video, a stream...

**Reply by dominicduffin1** [reply]
I like the idea of a connecting with community month. Or a Holiday season month of some kind. I also don't think I'll be participating in advent of code, I just don't feel I have the bandwidth for those kinds of challenges.

**Comment by ClJarvis** [comment]
A developer's holiday could be good. We're international it'd be good to see how people celebrate winter. Could maybe use the making-stuff channel. If you want alliteration Dev's December is right there.

**Comment by Andrew-Bush** [comment]
I think Dev December is a great name.  We have so many different cultures shared and we should remain agnostic, so it's perfect.  We also have folks in the southern hemisphere who are entering the summer season. 

@abbeyperini shared some great ideas around a spot-on theme for this month in [the general discussion here](https://github.com/Virtual-Coffee/VC-Contributors/discussions/144).  

#making-stuff is great partner channel to support the theme. 

I think it helps everyone not have to be working so hard for another month, but they also don't have to have idle hands.  I think we generally like to be busy and it's great to be busy with what we love to do and to be able to share that.

**Reply by abbeyperini** [reply]
I think maybe keep it in the monthly challenge channel and get real vague when describing making because Aurelie's crepe stream idea makes me think the food people could run with it and the content creation people could too.

If we're aiming for a break/refilling our bucket month and some of us will be doing winter holidays, perhaps the focus could be making self-care. We spend a lot of time making as programmers, so it's important to feed the fun, creativity aspect too.

Perhaps something along the lines of "Anything you make this month just for yourself. Whether it's a recipe you only make this time of year, a fun project you've been putting off, or solely self-expression. Share a photo or link and describe why you're happy you made it."

**Comment by ClJarvis** [comment]
December needs to be light weight recovery theme. So sharing something you'd already be doing sounds great.
